### PR TITLE
LSP: don't insert extra braces for "Wrap in element" auto-completions

### PR DESCRIPTION
**Pull request by jpnurmi**
Fixes the second issue mentioned in #3714: if the completion point is already followed by a "{", don't insert an additional "{}".

### CODE REVIEW

File: tools/lsp/language/completion.rs [hunk 1]
```diff
@@ -664,7 +672,11 @@ fn add_components_to_import(
             );
             result.push(CompletionItem {
                 label: format!("{} (import from \"{}\")", exported_name.name, file),
-                insert_text: Some(format!("{} {{$1}}", exported_name.name)),
+                insert_text: if has_brace {
+                    Some(format!("{}$1", exported_name.name))
```
Comment (ogoffart) [inline tools/lsp/language/completion.rs:676]: Actually, no need for $1 in that case. Just use a real thing
```suggestion
                    Some(exported_name.name.clone())
```

Comment (jpnurmi) [reply tools/lsp/language/completion.rs:676]: Ah, makes sense. Must be the default behavior.

**Comment by ogoffart** [comment]
Please also adjust line 76.

In this case, the insert_text can even stay none because by default it's the same as label.

**Comment by jpnurmi** [comment]
Done! Sorry for the delay, took a while to understand how to trigger this code path. Seems to be as simple as this:

```js
component Foo {
    | {}
}
```

**Review by ogoffart — APPROVED** [review]
Thanks.

A test would be ideal.

**Pull request merged**